batchflow/models/torch/blocks.py

```diff
@@ -298,3 +298,84 @@ def __init__(self, inputs=None, layout='nacd', filters=None, kernel_size=3, stri
+    split attention block that might have different (`radix`) and (`cardinality`) values that aimed to control the
+    number of groups within a second convolution (`radix`*`cardinality`), an attention part (`radix`) and RadixSoftmax.
+    Also, the amount of filters after block might be increasing by `external_mult` argument. Moreover, the block allows
+    to reduce number of filters inside the entire block via `bottleneck_width` or just inside an attention part via
+    `reduction_factor`.
+
+    The number of filters inside ResNeSt Attention calculates as following:
+    >>> filters = int(filters * (bottleneck_width / 64.)) * cardinality
+
+    The implementation was inspired by the authors' code (`<https://github.com/zhanghang1989/ResNeSt>`_), thus
+    the first 1x1 convolution is not split into groups, the second fully connected block does not contain
+    normalization. Constant 64 for `bottleneck_width` is also chosen by the authors.
+
+    Parameters
+    ----------
+    inputs : torch.Tensor
+        Example of input tensor to this layer.
+    layout : str
+        A sequence of letters, each letter meaning individual operation.
+        See more in :class:`~.layers.conv_block.BaseConvBlock` documentation. Default is 'cnScn'.
+    filters : int, str
+        If `str`, then number of filters is calculated by its evaluation. ``'S'`` and ``'same'`` stand for the
+        number of filters in the previous tensor. Note the `eval` usage under the hood.
+        If int, then number of filters in the output tensor. Default value is 'same'.
+    kernel_size : int, list of int
+        Convolution kernel size. Default is 3.
+    radix : int
+        The number of splits within a cardinal group. Default is 2.
+    cardinality : int
+        The number of feature-map groups. Given feature-map is splitted to groups with same size. Default is 1.
+    strides : int, list of int
+        Convolution stride. Default is 1.
+    reduction_factor : int
+        The number reflecting the size of the filter reduction in the inner layer. Default is 1.
+    bottleneck_width : int
+        The size of the reduction in the number of filters in the entire block. Default is 64.
+    attention : str
+        Name of self-attention module. For more info about possible operations,
+        check :class:`~.layers.SelfAttention`. Default is `sac`.
+    op : str or callable
+        Operation for combination shortcut and residual.
+        See more :class:`~.layers.Combine` documentation. Default is '+a'.
+    n_reps : int
+        Number of times to repeat the whole block. Default is 1.
+    kwargs : dict
+        Other named arguments for the :class:`~.layers.ConvBlock`.
+    """
+    def __init__(self, inputs=None, layout='cnaScn', filters='same', kernel_size=3, radix=2, cardinality=1,
+                 strides=1, reduction_factor=1, external_mult=4, bottleneck_width=64, attention='sac', op='+a',
```

Comment:
Undocumented `external_mult`, which suddenly changes the amount of desired filters: say, I passed `filters=64`, but for some reason, I get the tensor with `256` channels...